Automattic/wp-calypso
# CPT: Ensure "My Sites" master bar link is active when viewing list screen

This pull request seeks to resolve an issue where the "My Sites" master bar link is not active when viewing the [custom post types list screen](http://calypso.localhost:3000/types/post).

Because the fix was trivial (198e69f), I went ahead and bundled in a change for action types that have been bothering me and likely promoting poor action type naming to those observing the very-commonly-dispatched routing actions ([see guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#actions)).

__Testing instructions:__

Verify that My Sites link is active when viewing the [custom post types list screen](http://calypso.localhost:3000/types/post).

Verify that no instances of `SET_ROUTE` or `SET_SECTION` remain anywhere in the codebase.

Test live: https://calypso.live/?branch=fix/cpt-my-sites-active
